cartographer/cloud/internal/client/pose_graph_stub.cc

```diff
@@ -186,6 +186,10 @@ PoseGraphStub::constraints() const {
   return mapping::FromProto(client.response().constraints());
 }
 
+std::chrono::milliseconds PoseGraphStub::GetWorkQueueDelay() const {
+  return std::chrono::milliseconds(0);
```

Comment:
Add a `TODO` that this should be implemented?

Reply:
@wohe @CodeArno Adding this as a separate function to PoseGraphInterface seems a little one-offy. Is it not an option for you to use the metrics collection system to expose that information?

Reply:
Sounds like a good idea. @CodeArno Can you change the PR to use the metrics system directly?

Reply:
Good idea, done.